doc/api/test.md

```diff
@@ -1393,6 +1500,419 @@ added:
 This function is syntax sugar for [`MockTracker.method`][] with `options.setter`
 set to `true`.
 
+## Class: `MockTimers`
+
+<!-- YAML
+added:
+  - REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Mocking timers is a technique commonly used in software testing to simulate and
+control the behavior of timers, such as `setInterval` and `setTimeout`,
+without actually waiting for the specified time intervals.
+
+The [`MockTracker`][] provides a top level `timers` export
+which is a MockTimers instance
+
+### `timers.enable([,timers])`
+
+<!-- YAML
+added:
+  - REPLACEME
+-->
+
+Enables timer mocking for the specified timers.
+
+* `timers` {Array} An optional array containing the timers to mock.
+  The currently supported timer values are `'setInterval'` and `'setTimeout'`.
+  If no array is provided, all timers (`'setInterval'`, `'clearInterval'`, `'setTimeout'`,
+  and `'clearTimeout'`) will be mocked by default.
```

Comment:
This is somewhat ambiguous. Any default value should be a supported value.

Reply:
I think I didn't get it. Would you give more details?

Reply:
It says that the currently supported values are only `'setInterval'` and `'setTimeout'`, but the next sentence then says that by default, `'clear*'` will be mocked, too.